app/presenters/message_queue_presenter.rb

```diff
@@ -4,11 +4,11 @@ def self.present(downstream_presenter, update_type:)
       attributes = downstream_presenter.present
       # FIXME: This only happens in tests. We need to fix the tests so that
       # `link_set` is always available.
-      if downstream_presenter.link_set
-        links = Presenters::Queries::LinkSetPresenter.new(downstream_presenter.link_set).links
-      else
-        links = {}
-      end
+      links = if downstream_presenter.link_set
```

Comment:
There's been quite a lot of discussion over removing the cop that does this (Style/ConditionalAssignment). I think it hurts readability and looks ugly; we should probably disable it ourselves in rubocop.yml and revert the ones that got "fixed" here.